palo_alto_pan_os/komand_palo_alto_pan_os/actions/add_address_object_to_group/action.py

```diff
@@ -0,0 +1,61 @@
+import komand
+from .schema import AddAddressObjectToGroupInput, AddAddressObjectToGroupOutput, Input, Output, Component
+# Custom imports below
+from komand.exceptions import PluginException
+
+class AddAddressObjectToGroup(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='add_address_object_to_group',
+                description=Component.DESCRIPTION,
+                input=AddAddressObjectToGroupInput(),
+                output=AddAddressObjectToGroupOutput())
+
+    def run(self, params={}):
+        address_object = params.get(Input.ADDRESS_OBJECT)
+        group_name = params.get(Input.GROUP)
+        device_name = params.get(Input.DEVICE_NAME)
+        virtual_system = params.get(Input.VIRTUAL_SYSTEM)
+
+        # See if we can get the group the user is looking for:
+        xpath = f"/config/devices/entry[@name='{device_name}']/vsys/entry[@name='{virtual_system}']/address-group/entry[@name='{group_name}']"
+        response = self.connection.request.get_(xpath)
+
+        address_objects = response.get("response", {}).get("result", {}).get("entry", {}).get("static", {}).get(
+            "member")
+        if not address_objects:
+            raise PluginException(cause="PAN OS returned an unexpected response.",
```

Comment:
```suggestion
            raise PluginException(cause="PAN OS returned an unexpected response.",
```
Should PAN OS be swapped out for something else?